### ISSUE TITLE
결과보고서 최종 수정

### ISSUE THREAD

**Issue by wan500kim** [wan500kim/kickWorks#55]
기말 발표에 비해 아직 시간은 널널하지만 peer-review 받은 결과물을 수용하는 방면으로 미리 작성해 놓을 것을 제안합니다

**Comment by adap8709** [comment]
개발 일정으로 작성하였던 역할 배분을 올바르게 고치고,
프로젝트 개발환경에서 tensorflow 대신 실제 사용한 Keras로 바꾸는 등
일부 수정만 적용하면 될 것 같습니다.

**Comment by wan500kim** [comment]
네. 일정 및 역할 배분과 예시 사진 내의 필터링된 문장을 먼저 수정하고 일부 세부적 내용을 수정하도록 하겠습니다

**Comment by wan500kim** [comment]
제출 기한보다 깃헙 기한이 먼저이므로 닫지 않고 냅두도록 하겠습니다
